lib/block-supports/duotone.php

```diff
@@ -290,6 +290,59 @@ function gutenberg_tinycolor_string_to_rgb( $color_str ) {
 	}
 }
 
+class WP_Duotone {
```

Comment:
Not sure if there are some WordPress PHP style guidelines somewhere, but I always see classes in their own file named `class-wp-something.php`. Should we do the same for this class?

I also often see the Gutenberg version with a `_Gutenberg` suffix. In some places I think this is automatically added during the build process, but I don't think anything is done for `lib`. Do we need to include the suffix here so the name is `WP_Duotone_Gutenberg`?